crates/benchmark-runner/src/lib.rs

```diff
@@ -175,12 +171,12 @@ where
         None => panic!("unexpected test with no blocks {}", &bw.name),
     };
 
-    bw.blocks_and_witnesses = vec![last_block_with_witness];
+    let blocks_and_witnesses = vec![last_block_with_witness];
```

Comment:
Not sure if this vec with a single entry was done with a particular purpose originally, but I'm simply honoring the original style. If there's no good purpose of longer chain runs, we can refactor a bit and run only the latest blocks.

For now leaving it this way -- food for thought.

Reply:
Ah this was basically because with the execution spec bnchmarks, we only want the last block